lib/livebook/session.ex

```diff
@@ -1495,6 +1523,11 @@ defmodule Livebook.Session do
     put_in(state.memory_usage, %{runtime: runtime, system: Livebook.SystemResources.memory()})
   end
 
+  defp restore_secrets(state, runtime) do
```

Comment:
Let's make this `set_runtime_secrets(runtime, secrets)` and then we can use it in both places.